optimade/server/config.py

```diff
@@ -73,13 +73,16 @@ class ServerConfig(BaseSettings):
         ),
         description="Introspective information about this OPTIMADE implementation",
     )
+    index_base_url: Optional[AnyHttpUrl] = Field(
```

Comment:
Why do we have this now?

Reply:
We can remove it, sure, it's only (optionally) used in the landing page now

Reply:
Right. Is there a way for us to automagically retrieve it from the `links` endpoint data JSON file? Then there is only one source of truth and we're not mixing concepts. Especially, when it's using a recently outdated valid field name.